web/cobrands/buckinghamshire/js.js

```diff
@@ -0,0 +1,19 @@
+(function(){
+
+  if (!fixmystreet.maps) {
+      return;
+  }
+
+  $(function() {
+    $(document).on('change', '#form_speed_limit_greater_than_30', function(e) {
+      if ($(this).val() === 'no') {
+        // Report for the parish
+        fixmystreet.update_public_councils_text($('#js-councils_text').html(), [fixmystreet.bodies[1]]);
```

Comment:
I don't think we can rely on the ordering of fixmystreet.bodies. I think it will have to grep out Bucks here, and use Bucks in the other location.

Reply:
Ah OK, have changed it to look explicitly for the Bucks body and added some tests in 18722f83b84edd58cefb4a3a2b7f93ab7a8dd533.